xmeffects.js

```diff
@@ -56,7 +56,7 @@ function eff_t1_3(ch) {  // portamento
 
 function eff_t0_4(ch, data) {  // vibrato
   if (data & 0x0f) {
-    ch.vibratodepth = data & 0x0f;
+    ch.vibratodepth = (data & 0x0f) * 2;
```

Comment:
Probably depends on how they are scaling their internal period parameter. I will take your word for it here.